Pod/Classes/Acknowledgements/AcknowledgementViewController.swift

```diff
@@ -69,7 +69,7 @@ private extension AcknowledgementViewController {
         labelView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
             labelView.widthAnchor.constraint(equalTo: view.layoutMarginsGuide.widthAnchor),
-            labelView.topAnchor.constraint(equalTo: view.topAnchor),
+            labelView.topAnchor.constraint(equalTo: view.topAnchor, constant: 16.0),
```

Comment:
if it's inconsistent, we should probably add `.trimmingCharacters(in: .whitespacesAndNewlines)` (or maybe just `.newlines`) to the `cleanedUpLicense` variable and add reliable padding to the bottom as well